pkgs/build-support/docker/default.nix

```diff
@@ -935,33 +935,34 @@ rec {
           # following lines, double-check that your code behaves properly
           # when the number of layers equals:
           #      maxLayers-1, maxLayers, and maxLayers+1, 0
-          store_layers="$(
-            paths |
-              jq -sR '
-                rtrimstr("\n") | split("\n")
-                  | (.[:$maxLayers-1] | map([.])) + [ .[$maxLayers-1:] ]
-                  | map(select(length > 0))
-              ' \
-                --argjson maxLayers "$availableLayers"
-          )"
-
+          paths |
+            jq -sR '
+              rtrimstr("\n") | split("\n")
+                | (.[:$maxLayers-1] | map([.])) + [ .[$maxLayers-1:] ]
+                | map(select(length > 0))
+            ' \
```

Comment:
Maybe it has to be a new comment on a line that's from the latest state?

```suggestion
              ' \
```

Reply:
Nope. Maybe you haven't given permission for maintainers to push to your branch?